Automattic/wp-calypso
# Yoast: Adds upsell to stats and traffic pages

#### Proposed Changes

* Adds yoast upsell to `/stats` and `/marketing/traffic` pages.

#### Testing Instructions

* Navigate to stats page on a site without yoast premium installed and it should show the upsell.

![yoast-banner-stats](https://user-images.githubusercontent.com/140841/180866179-b49c1d85-48c3-4d27-8f1f-8a61c93dc8da.png)

* Navigate to traffic page on a site without yoast premium installed and it should show the upsell.

![yoast-banner-marketing-traffic](https://user-images.githubusercontent.com/140841/180866198-18ed0025-60f5-44f4-8edc-3f081fc6c555.png)

* Repeat the last two tests on a site with yoast installed, the upsell should not appear.
* Check that view and click events are being sent.



#### Pre-merge Checklist

Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.


Fixes #62977
